packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -165,7 +177,10 @@ const helpers = {
           httpContent
         );
       } else {
-        throw new Error('Requested content type is not defined in the schema');
+        return {
+          code,
+          mediaType: 'text/plain',
```

Comment:
If user requests a specific `mediaType` but we couldn't find it don't we want to return a problem+json response?

Reply:
Yes, but this is another story I think @StefanDywersant will tackle, right?

Reply:
If we just dont mention a mediaType will it default to text plan or will it try and JSON? I know fastify has some opinions.

Reply:
I can't do that, the `mediaType` is mandatory for the `IHttpNegotiationResult` (which is the returned object)